launch/upload.launch

```diff
@@ -15,10 +15,12 @@
 	<arg name="scale" default="1.0"/>
 	<arg name="side" default=""/>
 
+	<!-- extra xacro args required for melodic -->
+	<arg name="__noetic" value="$(eval env('ROS_PYTHON_VERSION') == '3')"/>
+	<arg name="__extra_args" value="" if="$(arg __noetic)"/>
+	<arg name="__extra_args" value="--xacro-ns" unless="$(arg __noetic)"/>
 	<!-- load robot description -->
-	<arg name="__noetic" value="$(eval env('ROS_PYTHON_VERSION') == '3')" />
-	<param name="robot_description" if="$(arg __noetic)" command="xacro $(find human_hand_description)/robots/human_hand.urdf.xacro use_synergy:=$(arg use_synergy) property_file:=$(arg property_file) mapping_file:=$(arg mapping_file) tactile_mapping:=$(arg tactile_mapping) tactile_channel:='$(arg tactile_channel)' prefix:=$(arg prefix) scale:=$(arg scale) side:=$(arg side)" />
-	<param name="robot_description" unless="$(arg __noetic)" command="xacro --xacro-ns $(find human_hand_description)/robots/human_hand.urdf.xacro use_synergy:=$(arg use_synergy) property_file:=$(arg property_file) mapping_file:=$(arg mapping_file) tactile_mapping:=$(arg tactile_mapping) tactile_channel:='$(arg tactile_channel)' prefix:=$(arg prefix) scale:=$(arg scale) side:=$(arg side)" />
+	<param name="robot_description" command="xacro $(arg __extra_args) $(find human_hand_description)/robots/human_hand.urdf.xacro use_synergy:=$(arg use_synergy) property_file:=$(arg property_file) mapping_file:=$(arg mapping_file) tactile_mapping:=$(arg tactile_mapping) tactile_channel:='$(arg tactile_channel)' prefix:=$(arg prefix) scale:=$(arg scale) side:=$(arg side)"/>
```

Comment:
From the visual diff in github, I see the `mapping_file` and `property_files` are already present in this separate commit

Reply:
Yes, of course. What is your point?

Reply:
it just seemed strange to have a separate commit to simplify robot_description upload and already mix it with the addition of the mapping file that occurs in the third commit. I sometimes also intend to nicely split commits (atomic commits) but make mistakes and include unwanted code. It was just to make you notice it.

Reply:
The mapping_file and property_file were introduced in the first commit, while the last one does the simplification.
As we should keep all three commits (to maintain authorship), I don't see an issue here.

Reply:
I apologized, I got confused about the order of commits in the github web interface for PR (not using it enough apparently).


For me github had a timeline in reverse commit (order) most recent top a seen on the main commit list 
![Screenshot 2022-10-19 at 11-41-57 Commits · ubi-agni human_hand](https://user-images.githubusercontent.com/6583181/196656879-c9918ae6-1d2b-43e2-acdc-d8994ce941d7.png)

but the PR does it reversed

![Screenshot 2022-10-19 at 11-42-31 Allow arbitrary property and mapping files by rhaschke · Pull Request #15](https://user-images.githubusercontent.com/6583181/196656754-0f1eb07e-5220-479d-b44d-a3c238494570.png)

For my defense, I should mention that there was no 4th commit yet, that could have helped me see the timeline is reversed with the date going up when going down the line

I saw the simplification as first (bottom) of the timeline, and made an interpretation from there. 

Thanks for making me discover how github is not consistent within its interfaces.